python/ribasim/ribasim/model.py

```diff
@@ -148,6 +149,47 @@ def _write_tables(self, directory: FilePath) -> None:
                 input_entry.write(directory, self.modelname)
         return
 
+    def validate_model(self):
+        """
+        Checks:
+        - Whether all node types in the node field are valid
+        - Whether the node IDs of the node_type fields are valid
+        """
+
+        node_names_all, node_cls_all = list(
+            zip(*inspect.getmembers(node_types, inspect.isclass))
+        )
```

Comment:
Maybe this should be a separate function/method somewhere?

Reply:
`inspect.getmembers` gives a list of tuples, I want two iterables with in one the names and in the other the classes

Reply:
I see.
Up to you if you want to put it into a function or not